src/webgpu/api/operation/render_pass/storeOp.spec.ts

```diff
@@ -1,23 +1,352 @@
 export const description = `API Operation Tests for RenderPass StoreOp.
 
-  Test Coverage Needed:
+  Test Coverage:
 
-  - Test that a render pass has correct output for combinations of:
-    - All color attachments from '0' to 'MAX_COLOR_ATTACHMENTS' with combinations of:
-      - storeOp set to {'clear', 'store', 'undefined}
-      - All color renderable formats
+  - Tests that color and depth-stencil store operations {'clear', 'store'} work correctly for a
+    render pass with both a color attachment and depth-stencil attachment.
+      TODO: Use depth24plus-stencil8
```

Comment:
nit: TODO also check that depthStoreOp and stencilStoreOp are separate.

Reply:
Added.